timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/MultiNodePaxosTimeLockServerIntegrationTest.java

```diff
@@ -64,6 +79,29 @@ public void bringAllNodesOnline() {
     }
 
     @Test
+    public void blockedLockRequestThrows503OnLeaderElection() throws InterruptedException {
```

Comment:
The build time has not increased significantly. Ran 6 builds, 2 of which flaked but in the CassandraIntegrationTests(old known flake).